app/AboutAppearanceViewController.m

```diff
@@ -158,4 +178,11 @@ - (IBAction)fontSizeChanged:(UIStepper *)sender {
     UserPreferences.shared.fontSize = @((int) sender.value);
 }
 
+- (void) setStatusBar:(id)sender {
+    UISwitch *switchy = sender;
+    [switchy setOn:switchy.on animated:YES];
```

Comment:
What does this do?

Reply:
I thought that this actually set the switch to another value on tap, could be wrong though... I don't understand how I can properly make one thing and then mess up on one of the more simpler concepts. I'll see whether it works without the code because I am a smooth brain some days, then if it does I will remove that code in my last commit and pray for forgiveness.